.doc_gen/metadata/bedrock-runtime_metadata.yaml

```diff
@@ -190,9 +190,17 @@ bedrock-runtime_InvokeStableDiffusion:
         - sdk_version: 3
           github: python/example_code/bedrock-runtime
           excerpts:
-            - description: Invoke the Stability.ai Stable Diffusion foundation model.
+            - description: Invoke the Stability.ai Stable Diffusion XL foundation model.
+              snippet_tags:
+                - php.example_code.bedrock-runtime.service.invokeStableDiffusion
```

Comment:
@DennisTraub could you double-check this metadata? It looks like the python version is showing PHP for StableDiffusion, while PHP is showing Meta Llama 2.